geoportal/tests/functional/test_functionalities.py

```diff
@@ -127,7 +128,7 @@ def test_functionalities(self):
                 "registered": {},
             },
             "admin_interface": {
-                "available_functionalities": ["__test_a", "__test_s"]
+                "available_functionalities": [{"name": "__test_a"}, {"name": "__test_s"}]
```

Comment:
Can it be something else than a name here ?

Reply:
Yes we can have: type, regex, error_message, or single, the more important one is single :-)

Reply:
Yes, we are in tests, but this is the definitions of functionalities.
My bad.